templates/csharp/api-plugin-from-scratch-bearer/.{{NewProjectTypeName}}/launchSettings.json.tpl

```diff
@@ -0,0 +1,9 @@
+{
+  "profiles": {
+    // Launch project within Teams
+    "Microsoft Teams (browser)": {
```

Comment:
Maybe we should change to `Copilot (browser)`

Reply:
All the launch related settings also should be changed to `Copilot (browser)`

Reply:
Sure.